packages/simple-markdown/src/index.js

```diff
@@ -746,11 +700,12 @@ var TABLES = (function () {
     var TABLE_CENTER_ALIGN = /^ *:-+: *$/;
     var TABLE_LEFT_ALIGN = /^ *:-+ *$/;
 
-    /**
-     * @param {string} alignCapture
-     * @returns {SimpleMarkdown.TableAlignment}
-     */
-    var parseTableAlignCapture = function (alignCapture) {
+    // TODO: This needs a real type
+    type TableAlignment = any;
```

Comment:
style nit: maybe use `$FlowFixMe` here so that it's easy to grep for types that need fixing.